README.md

```diff
@@ -59,8 +59,17 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 }
 ```
 
+
+Github SSH Authorization
```

Comment:
Does this also work for none github repositories? If so I would remove it from here.

```suggestion
SSH Authorization
```